spec/features/work_packages/datepicker/datepicker_non_working_day_spec.rb

```diff
@@ -45,11 +45,13 @@
     create(:non_working_day,
            date: Time.zone.today.beginning_of_week.next_occurring(:tuesday))
   end
+  shared_let(:working_day_this_week) { Time.zone.today.beginning_of_week.next_occurring(:wednesday) }
 
   shared_let(:non_working_day_next_year) do
     create(:non_working_day,
            date: Time.zone.today.end_of_year.next_occurring(:tuesday))
   end
+  shared_let(:working_day_next_year) { Time.zone.today.end_of_year.next_occurring(:wednesday) }
```

Comment:
Test will fail if year starts with a Wednesday
2031-01-01 is a Wednesday. I tried setting my clock to 2030 to see if the test fails and it fails: `expected to find visible field "work_package[due_date]" that is not disabled with value "2031-01-01" within #<Capybara::Node::Element tag="div" path="//HTML[1]/BODY[1]/OPCE-MODAL-OVERLAY[1]/DIV[2]/NG-COMPONENT[1]/DIV[1]/TURBO-FRAME[1]/DIV[1]"> but there were no matches. Also found "", which matched the selector but not all filters.` **`Expected value to be Wed, 01 Jan 2031 but was "2031-01-08"`**

I'll fix it.

Reply:
Fixed